samples/04_gis_analysts_data_scientists/information_extraction_from_cheshire_fire_incident_reports_using_mistral_language_model.ipynb

```diff
@@ -0,0 +1,1273 @@
+{
```

Comment:
<p>or with <s>larze </s>largelanguage model backbone</p><p> </p><p>&nbsp;extract entities from <s>te</s> the text.</p><p> </p><p>Spacy is not supported on the tool side.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>